README.rst

```diff
@@ -58,7 +63,7 @@ Manifest
     -  ``1`` if renewal not necessary;
     -  ``2`` in case of errors.
 
-12. ``--server`` (support multiple CAs).
+12. ``--server`` (support non-official ACME servers for custom CAs).
```

Comment:
there is no "official" ACME server; please revert this change or change wording; it's so happens to be that Let's Encrypt is a CA from the same folks that design the protocol